### PR TITLE
Community2

### PR DESCRIPTION
**Pull request by AibekYrysbekov**
[separate logic](https://github.com/AibekYrysbekov/airflow/commit/82e480e0d002d43e3bf38e145e93a4436e16e99c)

### CODE REVIEW

File: community/pmc-tool/main/collector/data_processor.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    insert_pull_requests_to_db(conn, prs=list(pr_authors_count.items()))
+    insert_issues_to_db(conn, issues=list(issue_authors_count.items()))
+
+    conn.close()
+
+    return pr_authors_count
+
+
+def read_reports():
```
Comment (pabloem) [inline community/pmc-tool/main/collector/data_processor.py:43]: it seems that this function is not called anywhere. Is that correct? If it's not called anywhere, we can remove it right?

File: community/pmc-tool/main/collector/queries.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+def verify_issues(conn, issue_authors_count):
+    issue_results = fetch_issues(conn)
+    assert_equal(issue_results, issue_authors_count)
+    return issue_results
+
+
+def get_first_pr_authors_last_week(conn):
+    return fetch_first_pr_authors_last_week(conn)
```
Comment (pabloem) [inline community/pmc-tool/main/collector/queries.py:24]: this is not called anywhere, right? can we remove it?

File: community/pmc-tool/main/collector/queries.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+def verify_pull_requests(conn, pr_authors_count):
+    pr_results = fetch_pull_requests(conn)
+    assert_equal(pr_results, pr_authors_count)
+    return pr_results
+
+
+def verify_issues(conn, issue_authors_count):
+    issue_results = fetch_issues(conn)
+    assert_equal(issue_results, issue_authors_count)
+    return issue_results
```
Comment (pabloem) [inline community/pmc-tool/main/collector/queries.py:20]: I think these make more sense in the verifiers file. Thoughts?

**Review by pabloem — COMMENTED** [review]
This is looking better.

This is just a question - not something that requires you to make a change - currently, the Github data will not be fetched, right? Because the function to download GH data is not called anywhere. Is that correct? This is okay for now - we can add a caller later - but can you confirm that my observation is correct?

File: community/pmc-tool/main/collector/queries.py (hunk 2 above)
Comment (AibekYrysbekov) [reply community/pmc-tool/main/collector/queries.py:24]: Retrieves the first pull request authors in the last week. called in views.py/ render_results

**Comment by pabloem** [comment]
LGTM

**Pull request merged**